CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ ENHANCEMENTS:
 * Added "cidrcontains" function. ([$366](https://github.com/opentofu/opentofu/issues/366))
 * Allow test run blocks to reference previous run block's module outputs ([#1129](https://github.com/opentofu/opentofu/pull/1129))
 * Support the XDG Base Directory Specification ([#1200](https://github.com/opentofu/opentofu/pull/1200))
+* Added support to resolve the issue of referencing output from a test run in the local variables block (tofu test). ([#1254](https://github.com/opentofu/opentofu/pull/1254))
```

Comment:
I think this reads a little better: WDYT?

```suggestion
* Allow referencing the output from a test run in the local variables block of another run (tofu test). ([#1254](https://github.com/opentofu/opentofu/pull/1254))
```

Reply:
Done. Thanks